backend/controllers/prisonerProfile/prisonerQuickLook.ts

```diff
@@ -130,7 +130,7 @@ export default ({
         incentivesApi.getIepSummaryForBooking(context, bookingId),
         prisonApi.getPositiveCaseNotes(context, bookingId, dateThreeMonthsAgo, today),
         prisonApi.getNegativeCaseNotes(context, bookingId, dateThreeMonthsAgo, today),
-        prisonApi.getAdjudicationsForBooking(context, bookingId),
+        adjudicationsApi.getAdjudicationsForBooking(context, bookingId),
```

Comment:
We have a future problem with this endpoint.  We don't store booking Id

Therefore we may need to either :
- (a) reinstate booking ID in the model (yuk!)
- (b) use the offender No. instead with maybe a cutoff date